### ISSUE TITLE
bug report: AppStudioPlugin.LocalAppIdUpdateFailed when adding localization to my manifest.source.json

### ISSUE THREAD

**Issue by FxWxP** [OfficeDev/TeamsFx#3026]
I am trying to add localization information to my manifest.source.json, I am following the instructions from: https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/apps-localization#localize-strings-in-your-app-manifest
I have added the following code to my manifest.source.json:
```
    "localizationInfo": {
		"defaultLanguageTag": "en",
		"additionalLanguages": [
			{
				"languageTag": "nl",
				"file": "nl.json"
			}
		]
	}
```
and added a file in the same folder called nl.json
```
{
    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.Localization.schema.json",
    "name.short": "Dutch short name"
}
```

How ever if I start my app locally, I get the following error and stack trace. I couldn't find any documentation about how to do this for TeamsFx so I might have made a mistake.

```
Failed to update local app id in app studio due to Error: Request failed with status code 500.

stack:
LocalAppIdUpdateFailed: Failed to update local app id in app studio due to Error: Request failed with status code 500.
	at Function.SystemError (<REDACTED: user-file-path>:2:5660434)
	at AppStudioPluginImpl.updateApp (<REDACTED: user-file-path>:2:5654890)
	at runMicrotasks (<anonymous>)
	at processTicksAndRejections (<REDACTED: user-file-path>:93:5)
	at async AppStudioPluginImpl.getAppDefinitionAndUpdate (<REDACTED: user-file-path>:2:5611170)
	at async AppStudioPluginImpl.postLocalDebug (<REDACTED: user-file-path>:2:5632407)
	at async AppStudioPlugin.postLocalDebug (<REDACTED: user-file-path>:2:5603713)
	at async Promise.all (index 5)
	at async Object.executeConcurrently (<REDACTED: user-file-path>:2:6268792)
	at async TeamsAppSolution.doLocalDebug (<REDACTED: user-file-path>:2:6312152)
	at async TeamsAppSolution.localDebug (<REDACTED: user-file-path>:2:6309420)
	at async FxCore.localDebug (<REDACTED: user-file-path>:2:5042850)
	at async FxCore.exports.LocalSettingsWriterMW (<REDACTED: user-file-path>:2:5089202)
	at async FxCore.<anonymous> (<REDACTED: user-file-path>:2:5082757)
	at async FxCore.exports.ProjectSettingsWriterMW (<REDACTED: user-file-path>:2:5124325)
	at async FxCore.exports.ContextInjectorMW (<REDACTED: user-file-path>:2:5073482)
	at async FxCore.exports.QuestionModelMW (<REDACTED: user-file-path>:2:5135776)
	at async FxCore.<anonymous> (<REDACTED: user-file-path>:2:5136675)
	at async FxCore.exports.LocalSettingsLoaderMW (<REDACTED: user-file-path>:2:5088743)
	at async FxCore.<anonymous> (<REDACTED: user-file-path>:2:5079987)
	at async FxCore.exports.ProjectSettingsLoaderMW (<REDACTED: user-file-path>:2:5123117)
	at async FxCore.exports.ProjectUpgraderMW (<REDACTED: user-file-path>:2:5132132)
	at async FxCore.exports.ProjectMigratorMW (<REDACTED: user-file-path>:2:5120230)
	at async FxCore.exports.ConcurrentLockerMW (<REDACTED: user-file-path>:2:5072627)
	at async FxCore.exports.ErrorHandlerMW (<REDACTED: user-file-path>:2:5085041)

```

**Comment by nliu-ms** [comment]
@FxWxP  Thank you for reaching out! Let me have some investigations and get back to you ASAP.

**Comment by nliu-ms** [comment]
It has been fixed in Teams Toolkit v3.1.0, please update and test if it works for you @FxWxP . Thank you!

**Comment by FxWxP** [comment]
It works like a charm, both on local dev as well as deployed to the cloud. 
Thanks for this great christmas gift!
